src/Couchbase.Lite.Shared/API/Query/IMetaExpression.cs

```diff
@@ -35,7 +35,7 @@ public interface IMetaExpression : IExpression
         /// <param name="alias">The name of the data source</param>
         /// <returns>The expression with the alias added</returns>
         [NotNull]
-        IExpression From(string alias);
+        IExpression From([NotNull]string alias);
```

Comment:
Missing contract annotation

Reply:
We decide to remove all contract annotation attributes from entire code base.